test/e2e/app-dir/app-prefetch/prefetching.test.ts

```diff
@@ -114,51 +114,6 @@ describe('app dir - prefetching', () => {
     ).toBe(1)
   })
 
-  it('should not fetch again when a static page was prefetched when navigating to it twice', async () => {
```

Comment:
these tests were moved into `prefetching.staletimes.test.ts` since they're no longer valid when the staletime values are left unconfigured.